src/main/drivers/accgyro/accgyro.h

```diff
@@ -78,6 +79,13 @@ typedef enum {
     GYRO_RATE_32_kHz,
 } gyroRateKHz_e;
 
+typedef enum {
+    INIT = 0,
+    EXTI_INT_DMA,
+    EXTI_INT,
+    NO_EXTI_INT
+} gyroModeSPI_e;
+
```

Comment:
These should have a suitable prefix on them, to avoid clashes with other local enums using the same names, e.g. `INIT` is very generic.  Perhaps:

```
typedef enum {
    GYRO_MODE_INIT = 0,
    GYRO_MODE_EXTI_INT_DMA,
    GYRO_MODE_EXTI_INT,
    GYRO_MODE_NO_EXTI_INT
} gyroModeSPI_e;
```

Reply:
@SteveCEvans This still needs addressing.  When I merge with some of my other PR's I get compile errors because of `INIT` being too generic - it's also used locally in my code in a different file.